docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+# Converting boot-from-image servers to boot-from-volume
+
+You can freely move any server [between {{brand}}
+regions](move-server-between-regions), provided it uses
+boot-from-volume. Since such servers offer much more flexibility than
+those using boot-from-image, we recommend the former over the latter.
+You may still have boot-from-image servers, though. If you decide to
+move any of those between regions, you will discover that you cannot do
+so. The solution is first to convert them to boot-from-volume servers
+and then move them. In the following, we show how to perform the
+conversion using the {{gui}} or OpenStack CLI.
+
+## Preparation
+
+=== "{{gui}}"
+    Fire up your favorite web browser and navigate to the
+    [{{gui}}](https://{{gui_domain}}) start page. Log into your {{brand}}
+    account if you have to.
+=== "OpenStack CLI"
+    To work with the OpenStack CLI, make sure to properly [enable
+    it](/howto/getting-started/enable-openstack-cli) for the region your
+    boot-from-image server resides in.
+
+## Shutting down the server
+
+=== "{{gui}}"
+    Make sure the left-hand side vertical pane is in full view, then select
+    Compute → [Servers](https://{{gui_domain}}/compute/servers). You will
+    see all supported {{brand}} regions in the main pane. Expand the one
+    with the boot-from-image server you want to convert to boot-from-volume.
+    Click the orange :fontawesome-ellipsis-circle: icon at the right of the
+    server row, and from the pop-up menu that appears, select *Stop Server*.
+
+    ![Stopping server](assets/bfi-to-bfv/shot01.png)
+
+    A pop-up window appears, asking if you want to stop the server. Confirm
+    by clicking on the red button labeled *Yes, Stop*.
+=== "OpenStack CLI"
+    Assuming your boot-from-image server is named
+    [`nanisivik`](https://en.wikipedia.org/wiki/Nanisivik), using your
+    favorite terminal application, shut it down by typing the following:
+
+    ```bash
+    openstack server stop nanisivik
+    ```
+
+    To confirm that it has finished shutting down, type:
+
+    ```bash
+    openstack server show nanisivik -c status -f plain
```

Comment:
`-f plain` is not a valid option for the `openstack` CLI. 
```suggestion
    openstack server show nanisivik -c status -f value
```

Reply:
¡Dios mío! (Good catch!)